monitor_frontend/views/events_table.view.lkml

```diff
@@ -0,0 +1,111 @@
+include: "//looker-hub/monitor_frontend/views/events.view.lkml"
+
+view: +events {
```

Comment:
I'd recommend refining the underlying `events_unnested_table` hub view instead (plus renaming this file `events_unnested_table.view.lkml`):
```suggestion
include: "//looker-hub/monitor_frontend/views/events_unnested_table.view.lkml"

view: +events_unnested_table {
```

However, if you stick with refining the `events` hub view then this file should be named `events.view.lkml`.